nmostesting/suites/IS1101Test.py

```diff
@@ -45,34 +48,33 @@ def __init__(self, apis):
         self.caps = ""
 
     # GENERAL TESTS
-    def test_00_01(self, test):
+    def test_01(self, test):
         """Verify that IS-11 is exposed in the Node API as \
         urn:x-nmos:control:stream-compat/v1.0 at url /x-nmos/streamcompatibility/v1.0/
         """
-        valid_res, response = TestHelper.do_request(
-            "GET", self.base_url + "/x-nmos/node/v1.3/devices/"
+        _, response = TestHelper.do_request(
+            "GET", self.node_url + "devices/"
         )
-        if valid_res:
-            response_json = response.json()
-            controls = response_json[0][CONTROLS]
-            control_href = ""
+        if response.status_code != 200:
+            return test.FAIL("The request has not succeeded.", response)
+        controls = response.json()[0][CONTROLS]
+        control_href = ""
+        if len(controls) > 0:
             for control in controls:
-                if control["type"] == "urn:x-nmos:control:stream-compat/v1.0":
+                if control["type"] == "urn:x-nmos:control:stream-compat/" + self.apis[COMPAT_API_KEY]["version"]:
                     control_href = control["href"]
                     break
-            if len(control) == 0:
-                return test.WARNING("IS-11 API is not available")
             if not control_href.endswith(self.compat_url):
                 return test.FAIL("IS-11 URL is invalid")
             return test.PASS()
-        return test.FAIL(response)
+        return test.FAIL("IS-11 API is not available")
 
-    def test_00_02(self, test):
+    def test_02(self, test):
         "Put all senders into inactive state"
-        senders_url = self.base_url + "/x-nmos/connection/v1.1/single/senders/"
+        senders_url = self.conn_url + "single/senders/"
         _, response = TestHelper.do_request("GET", senders_url)
         if response.status_code != 200:
-            return test.FAIL(response.json())
+            return test.FAIL("The request has not succeeded.", response.json())
```

Comment:
I don't think these are valid arguments to `FAIL`...

https://github.com/AMWA-TV/nmos-testing/blob/aa8a2317c15e3354408830e1f0b2b48fd1db111a/nmostesting/TestResult.py#L98